subliminal/providers/argenteam.py

```diff
@@ -113,21 +109,22 @@ def search_episode_id(self, series, season, episode):
         """
         # make the search
         query = '%s S%#02dE%#02d' % (series, season, episode)
-        logger.info('Searching show id for %r', query)
-        r = self.session.get(self.API_URL + 'search',
-                             params={'q': query}, timeout=10)
+        logger.info('Searching episode id for %r', query)
+        r = self.session.get(self.API_URL + 'search', params={'q': query}, timeout=10)
         r.raise_for_status()
         results = json.loads(r.text)
         episode_id = None
         if results['total'] == 1:
             episode_id = results['results'][0]['id']
+        else:
+            logger.error('No show id found for %r', series)
```

Comment:
I think you mean episode_id here.
Also the docstring of that function is not accurate.